ansible/support/tasks/ignition.yml

```
@@ -2,6 +2,25 @@
 - hosts: all
 
   tasks:
+  - name: Wait for all MCPs to be stable before fetching ignition
+    shell: |
+      echo "Checking MCP stability..."
+      for mcp in $(oc get mcp -o jsonpath='{.items[*].metadata.name}'); do
+        echo "Waiting for MCP: $mcp"
+        oc wait --for=condition=Updated=True --timeout=30m mcp/$mcp || true
+        oc wait --for=condition=Updating=False --timeout=30m mcp/$mcp || true
+        oc wait --for=condition=Degraded=False --timeout=5m mcp/$mcp || true
+      done
+      echo "All MCPs are stable"
+    register: mcp_wait_result
+    retries: 3
+    delay: 30
+    until: mcp_wait_result.rc == 0
+
+  - name: Additional stabilization wait for MCS endpoints
```

```suggestion
  - name: Additional stabilization wait for Machine Config Server endpoints
```

`Machine Config Server`

this adds another 2 minutes.